### PR TITLE
replaced product image and product video helper with facades

### PR DESCRIPTION
**Pull request by prateek-webkul**

### CODE REVIEW

**Review by rahulshukla-webkul — CHANGES_REQUESTED** [review]
Hi @prateek-webkul 

Kindly create helpers for each facades.

**Comment by prateek-webkul** [comment]
> Hi @prateek-webkul
> 
> Kindly create helpers for each facades.

Implemented

File: packages/Webkul/Product/src/Providers/ProductServiceProvider.php [hunk 1]
```diff
@@ -79,4 +88,29 @@ protected function registerEloquentFactoriesFrom($path): void
...
+        // Product image
+        $loader = AliasLoader::getInstance();
+        $loader->alias('productimage', ProductImageFacade::class);
+
+        $this->app->singleton('productimage', function () {
+            return app()->make(ProductImage::class);
+        });
+
+        // Product video
+        $loader = AliasLoader::getInstance();
```
Comment (rahulshukla-webkul) [inline packages/Webkul/Product/src/Providers/ProductServiceProvider.php:109]: No need to declare $loader two times.

Comment (prateek-webkul) [reply packages/Webkul/Product/src/Providers/ProductServiceProvider.php]: removed another one

**Review by rahulshukla-webkul — APPROVED** [review]

**Pull request merged**